src/main/java/com/ekart/config/SecurityConfig.java

```diff
@@ -50,12 +51,12 @@ protected void configure(HttpSecurity http) throws Exception {
 
         http.authorizeRequests()
                 .antMatchers("/login").permitAll()
-//                .antMatchers(HttpMethod.DELETE).hasRole("ADMIN")
-//                .antMatchers("/api/v1/brands/**").hasAnyRole("ADMIN", "STAFF")
-//                .antMatchers("/api/v1/category/**").hasAnyRole("ADMIN", "STAFF")
-//                .antMatchers("/api/v1/sizes/**").hasAnyRole("ADMIN", "STAFF")
-//                .antMatchers(HttpMethod.POST, "/api/v1/products/**").hasAnyRole("ADMIN", "STAFF")
-//                .antMatchers(HttpMethod.PUT, "/api/v1/products/**").hasAnyRole("ADMIN", "STAFF")
+                .antMatchers(HttpMethod.GET).permitAll()
```

Comment:
i'm changing the /products Get method to Post due to same common method for all filters and sort conditions

Reply:
I have added this